pairing/bn256/README.md

```diff
@@ -0,0 +1,18 @@
+bn256
+-----
+
+Package bn256 implements a particular bilinear group at the 128-bit security
+level. The basis for this package is the code from
+https://github.com/cloudflare/bn256 which itself is a modification of the
```

Comment:
Why is this copied rather than vendored? That is, what are your modifications?

Reply:
Thanks for your comments! I've copied everything over since I wasn't sure what modifications I would need to do. Now that I have a better overview, I think we can use your library directly using the wrappers I wrote. I will do the necessary tweaks next week.

Reply:
@Bren2010: To answer your question on the modifications, I replaced the high-level API in [bn256.go](https://github.com/cloudflare/bn256/blob/master/bn256.go) with implementations of the Kyber interfaces [point](https://github.com/dedis/kyber/blob/pairing/pairing/bn256/point.go), [scalar](https://github.com/dedis/kyber/blob/pairing/pairing/bn256/scalar.go), [group](https://github.com/dedis/kyber/blob/pairing/pairing/bn256/group.go), and [suite](https://github.com/dedis/kyber/blob/pairing/pairing/bn256/suite.go). I did not touch the lower level files except for `gfp12.go` where I added the [neutral element of GT](https://github.com/dedis/kyber/blob/pairing/pairing/bn256/gfp12.go#L48-L77).

For now we probably need to keep a copy of your library within Kyber because we need access to several of the unexported structs and constants to implement the above interfaces. This includes `curvePoint`, `twistPoint`, `gfP12`, `curveGen`, `twistGen`, `gfP12Inf`, `gfP12Gen` and probably a few others that I have overlooked. I could write wrappers around your exported points G1, G2, GT etc. but that might become quite confusing and even then I am not sure we can access all the functionality we need. What do you think?

Reply:
Sounds good to me. I'd recommend putting this info in the readme.

Reply:
Good idea, will do that.